ytypes/list.go

```diff
@@ -271,11 +273,12 @@ func nameMatchesPath(fieldName string, path []string) (bool, error) {
 
 // unmarshalList unmarshals a JSON array into a list parent, which must be a
 // map or slice ptr.
-//   schema is the schema of the schema node corresponding to the struct being
-//     unmamshaled into
-//   jsonList is a JSON list
-//   opts... are a set of ytypes.UnmarshalOptionst that are used to control
-//     the behaviour of the unmarshal function.
+//
+//	schema is the schema of the schema node corresponding to the struct being
+//	  unmamshaled into
+//	jsonList is a JSON list
+//	opts... are a set of ytypes.UnmarshalOptionst that are used to control
+//	  the behaviour of the unmarshal function.
 func unmarshalList(schema *yang.Entry, parent interface{}, jsonList interface{}, enc Encoding, opts ...UnmarshalOpt) error {
```

Comment:
Change to a [list](https://go.dev/doc/comment#lists) instead of the current erroneous indenting.